libraries/c_sdk/aws/defender/test/system/aws_iot_tests_defender_system.c

```diff
@@ -760,3 +806,28 @@ static void _verifyTcpConnections( int total )
 }
 
 /*-----------------------------------------------------------*/
+
+static Socket_t _createSocketToEchoServer()
+{
+    static const TickType_t xReceiveTimeOut = pdMS_TO_TICKS( 2000 );
+    static const TickType_t xSendTimeOut = pdMS_TO_TICKS( 2000 );
```

Comment:
Why these variables are declared static ?

Reply:
Not quite sure either. They were originally declared static before the separation PR.

Reply:
It could be cut and paste error,  they need not be static.